.buildkite/pipeline.yml

```diff
@@ -0,0 +1,28 @@
+docker-container: &docker-container
+  plugins:
+    - docker#v3.8.0:
+        image: "wordpressmobile/android-build-image"
+        environment:
+          - "CI=true"
```

Comment:
I'm surprised this is necessary given https://buildkite.com/docs/pipelines/environment-variables#bk-env-vars-ci, but not a huge thing.

Reply:
@jkmassel I can test without it, but I don't think it'd be passed to the Docker container unless we say:

```
environment:
  - "CI"
```

And if we are going to already add this, we might as well set it to `true` since it carries more information at a glance.

Reply:
I am going to merge this for now and will report back on my test later.

Reply:
@jkmassel I've tried this in `try/docker-without-ci-env`, and as I expected, the build did not publish the Gradle scan.

_P.S: I am not linking the job here because it's not accessible by public._